tests/spec/core/validators-spec.js

```diff
@@ -0,0 +1,96 @@
+import {
+  validateCommonName,
+  validateDOMName,
+  validateMimeType,
+} from "../../../src/core/dfn-validators.js";
+
+describe("Core - Validators", () => {
+  describe("validateDOMName", () => {
+    it("doesn't generate an error if the element is valid", () => {
+      const elements = [
+        "h1",
+        "feSpecularLighting",
+        "feGaussianBlur",
+        "body",
+        "html",
+      ];
+      for (const element of elements) {
+        const dfn = document.createElement("dfn");
+        expect(validateDOMName(element, "element", dfn, "foo/bar")).toBeTrue();
```

Comment:
I came across jasmine's `withContext`, which lets us add some context to `expect` failure messages. Could be useful when testing in a loop.

```suggestion
        expect(validateDOMName(element, "element", dfn, "foo/bar"))
          .withContext(`element name: ${element}`)
          .toBeTrue();
```

![image](https://user-images.githubusercontent.com/8426945/126461646-c838eca4-bcb9-4f57-9be0-6600818e9287.png)

Reply:
neat!

Reply:
Let's do it on all new tests?

Reply:
yep. Done.